nevergrad/optimization/optimizerlib.py

```diff
@@ -437,7 +437,15 @@ def _internal_tell(self, x: ArrayLike, value: float) -> None:
             self.evaluated_population_sigma = []
             self.evaluated_population_fitness = []
 
+            
+# We have seen that NaiveTBPSA is a great extension of TBPSA for the noise-free case.
+# ARF1 suggested that something similar could happen for MPCEDA.
```

Comment:
Not sure this comment is clear for everyone, change this before approving

Reply:
Also, this is probably far too generous towards me. I was just complaining that the optimizer did not seem to work as I had expected - while apparently misunderstanding that the optimizer was indeed working as it should. Thanks for being so responsive though!

Reply:
@arf1: is MPCEDA or NaiveMPCEDA good for you ? If yes that's a good reason for adding it.

Reply:
@teytaud I have not tried it yet but I will give it a try.

As I am not an expert in optimization I naively ran all optimizers against my problem (NN reinforcement learning) to see which came out on top. I will give NaiveMPCEDA a try. Though from what I have seen so far, I expect other algorithms are better suited.

Reply:
@teytaud For me NaiveMPCEDA is better than MPCEDA but neither is really well-suited to my problem. I get the best results (absolute result as well as result vs. speed) with CMA-variants and DE-variants.

Reply:
thanks @arf1